scripts/generate_readme.rb

```diff
@@ -16,7 +16,7 @@ def build
 
       This repository contains the core code base and documentation for the identity management system powering secure.login.gov.
 
-      **This file is auto-generated**, to edit it, check out [`#{__FILE__}`](#{__FILE__})
+      **This file is auto-generated** by [`#{__FILE__}`](#{__FILE__}). To edit it, run `make README.md`.
```

Comment:
I like the inclusion of `make README.md`, but edits to the contents of `README.md` itself would have to be made in this file (`scripts/generate_readme.rb`), which I think is what the previous text was referring to, and the only reason to mention `scripts/generate_readme.rb`. I worry "edit it" could now be a bit ambiguous.

Maybe we could update the phrasing to describe what `make README.md` does, and/or drop the reference to `scripts/generate_readme.rb` . Something like...

```suggestion
      **This file is auto-generated**. Run `make README.md` to regenerate its contents.
```